src/xcode/ENA/ENA/Source/Services/CCLService/__tests__/CCLConfiguationTests.swift

```diff
@@ -171,7 +188,16 @@ class CCLServiceConfigurationTests: CCLServiceBaseTests {
 		)
 		let cache = try cache(with: Locator.CCLConfiguration(isFake: false), eTag: eTag, date: yesterday, responseData: oldCCLConfigurationData)
 		let restServiceProvider = RestServiceProvider(session: stack.urlSession, cache: cache)
-		let cclService = CCLService(restServiceProvider, appConfiguration: appConfiguration, cclServiceMode: [.configuration], signatureVerifier: MockVerifier())
+		let resource = MockCCLConfigurationResource(defaultModel: nil)
```

Comment:
Why not use "CCLConfigurationResource" directly and overwriting the default model (property) after initialization?

Reply:
Done that now 👍